packages/apptive_grid_core/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: apptive_grid_core
 description: Core Library for ApptiveGrid used to provide general ApptiveGrid functionality to other Packages or Apps
-version: 0.13.3
+version: 0.14.0
```

Comment:
As this is a non breaking change I think that `0.13.4` is a more sensible new version

Reply:
done